lib.rs

```diff
@@ -233,7 +233,10 @@ impl slog::Drain for StdLog {
             slog::Level::Trace => log::LogLevel::Trace,
         };
 
-        let target = info.tag();
+        let mut target = info.tag();
+        if target.is_empty() {
+            target = info.module();
+        }
 
         let location = log::LogLocation {
             __module_path: info.module(),
```

Comment:
We already are passing module information to `log`. Does anyone know what is the rationale for `log` to set `target` to be `module` as default?

Reply:
When unspecified, non-slog `log!` sets `target` to the module. This is aligning the default values the record takes when calling slog `log!(logger, "spam")` and non-slog `log!("spam")`